### PR TITLE
chore(build): cleanup workflow and split lint to other workflow

### PR DESCRIPTION
**Pull request by jsdidierlaurent**

### CODE REVIEW

File: .github/workflows/linter.yml [hunk 1]
```diff
@@ -0,0 +1,64 @@
+---
+name: chmod-checker
```
Comment (Alex-D) [inline .github/workflows/linter.yml]: lint?

and rename file `lint`

Comment (jsdidierlaurent) [reply .github/workflows/linter.yml]: done ^^

File: ui/package.json [hunk 2]
```diff
@@ -4,7 +4,7 @@
   "scripts": {
     "serve": "vue-cli-service serve",
     "build": "vue-cli-service build",
-    "test": "npm-run-all lint",
+    "test": "echo TODO",
```
Comment (Alex-D) [inline ui/package.json]: `npm-run-all test:unit` even if it does not work

Comment (jsdidierlaurent) [reply ui/package.json]: done

**Pull request merged**